test_infra/test_infra/base_stack.py

```diff
@@ -0,0 +1,129 @@
+from aws_cdk import aws_ec2 as ec2
```

Comment:
Minor one, can we rename the directory to "stacks" instead of "test_infra"? Just to avoid having two directories named "test_infra"

Reply:
I was trying to follow the usual naming standards for python packages here e.g. there's the root of the repo, and then python package inside with very often the same name. I'm all right to change it, though.

Reply:
Happy with convention 👍🏼